package.json

```diff
@@ -43,7 +43,7 @@
     "supertest": "^4.0.2"
   },
   "engines": {
-    "node": "10.15.0",
+    "node": "^12.13.1",
```

Comment:
I made this a fuzzy match so that it is OK to use future patch releases of 12.13.